pkg/dockerregistry/server/admin.go

```diff
@@ -46,9 +47,12 @@ func (bh *blobHandler) Delete(w http.ResponseWriter, req *http.Request) {
 
 	err := bh.Registry().Blobs().Delete(bh.Digest)
 	if err != nil {
-		bh.Errors.PushErr(fmt.Errorf("error deleting blob %q: %v", bh.Digest, err))
-		w.WriteHeader(http.StatusBadRequest)
-		return
+		// Ignore PathNotFoundError
+		if _, ok := err.(storagedriver.PathNotFoundError); !ok {
```

Comment:
Maybe, but this code is ultimately going away when we move pruning of registry-owned content to the registry itself. So I'm ok foregoing for now.